capitalize.rb

```diff
@@ -0,0 +1,7 @@
+require_relative 'base_decorator'
+
+class CapitalizeDecorator < Decorator
+  def correct_name
+    @nameable.correct_name.upcase
+  end
+end
```

Comment:
- [ ]  Great job so far @CesarHerr There are just a few changes I will suggest you implement and you'll be on your way to the next milestone :+1: 
- [ ] Per the project requirements, your `CapitalizeDecorator` class is needed to Implement a method `correct_name` that capitalizes the output of `@nameable.correct_name`. This means only the first letter of the given name needs to be capitalized and not the entire name so kindly fix that :wink: 

- Your Current Output
![image](https://github.com/CesarHerr/OOP_school_library/assets/102699958/14dcffd8-e477-4bdb-b9a2-2a206b2d5688)

Reply:
@NahnahAJ  I appreciate your review! Understood, I will address this promptly :rocket: